src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+// import React from 'react';
```

Comment:
Great design @ijeomaodinko 🤝

- [ ] [Optional] Always remove commented code from your project as this is not regarded as good practice.